flowmachine/flowmachine/features/subscriber/distance_series.py

```diff
@@ -0,0 +1,140 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# -*- coding: utf-8 -*-
+"""
+Per subscriber time series of distances from some reference location.
+"""
+from typing import List, Optional
+
+from flowmachine.features.spatial import DistanceMatrix
+from .metaclasses import SubscriberFeature
+from ..utilities.subscriber_locations import SubscriberLocations, BaseLocation
+
+
+valid_stats = {"sum", "avg", "max", "min", "median", "stddev", "variance"}
+valid_time_buckets = ["second", "minute", "hour", "day", "month", "year", "century"]
```

Comment:
Is `"week"` deliberately missing? Just checking - I can see that it is a slightly ambiguous concept and we may not want to include it because of this.